src/eval/render_helpers.zig

```diff
@@ -221,6 +221,52 @@ pub fn renderValueRocWithType(ctx: *RenderCtx, value: StackValue, rt_var: types.
                         try out.append(']');
                         return out.toOwnedSlice();
                     }
+                    // Special handling for Dict - render as {key: value, ...}
```

Comment:
I think we should actually render this as `Dict.from_list([(k1, v1), (k2, v2)])`